docs/docker-image.md

```diff
@@ -0,0 +1,54 @@
+# `scan-docker-image` (experimental)
+This `vault-radar` command is used for scanning a docker image
+
+## Usage
+
+### Scanning a docker image
+
+Scan a public docker image (or a private image that is already pulled/dowloaded locally) and write the results to a file in CSV format, this is the default format for output. Image reference may optionally include a tag.[Docker engine](https://docs.docker.com/engine/install/) is a pre-requisite for scanning docker images using vault-radar.
```

Comment:
Should we explicitly call out that we will attempt to scan latest tag if one isn't specified? Also, it isn't mandatory to have a latest tag